controllers/profiles/status_enricher_dev_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package profiles
+
+import (
+	"context"
+	"testing"
+
+	openshiftv1 "github.com/openshift/api/route/v1"
+	"github.com/stretchr/testify/assert"
+	"knative.dev/pkg/apis"
+
+	apiv08 "github.com/kiegroup/kogito-serverless-operator/api/v1alpha08"
+	"github.com/kiegroup/kogito-serverless-operator/test"
+)
+
+func Test_enrichmentStatusOnK8s(t *testing.T) {
+	t.Run("verify that the service URL is returned with the default cluster name on default namespace", func(t *testing.T) {
+
+		workflow := test.GetKogitoServerlessWorkflow("../../config/samples/"+test.KogitoServerlessWorkflowSampleDevModeYamlCR, t.Name())
+		workflow.Namespace = "test"
```

Comment:
```suggestion
		workflow.Namespace = t.Name()
```
So you won't mess with other tests running in parallel.